application-confluence-migrator-pro-ui/src/main/resources/ConfluenceMigratorPro/Code/CommonCode.xml

```diff
@@ -508,14 +524,27 @@
   });
 
   $(function() {
-    // Since the associated liveData is refreshed on file upload, we don't need a response container.
-    const input = document.querySelector('#addConfluencePackage input[type=file]');
-    if (input &amp;&amp; XWiki.FileUploader) {
-      new XWiki.FileUploader(input, {
-        'progressAutohide': true,
-        'responseContainer' : document.createElement('div')
-      });
-    }
+    const input = document.querySelector('#confluenceUploadFile');
+
+    if (!input || !XWiki.FileUploader) return;
+
+    new XWiki.FileUploader(input, {
+      progressAutohide: true,
+      responseContainer: document.createElement('div'),
+    });
+
+    input.observe('xwiki:actions:beforeUpload', function(event) {
+      const file = input.files[0];
+
+      if (!file || !file.name.toLowerCase().endsWith('.zip')) {
+        event.preventDefault();
+        input.value = '';
+        new XWiki.widgets.Notification(
+          `It looks like you are trying to import a file of an unexpected format. Confluence Migrator Pro currently only supports Confluence backup exports, which have a .zip extension.`,
```

Comment:
We will probably need to have this string translated at some point, it would be nice to use the translation api here